<ui version="4.0" >
 <class>YoutubeDlOption</class>
 <widget class="QWidget" name="YoutubeDlOption" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>349</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="fileSettingGroupBox" >
     <property name="title" >
      <string>File Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3" >
      <property name="fieldGrowthPolicy" >
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="fileNameLabel" >
        <property name="text" >
         <string>Filename:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KComboBox" name="fileNameComboBox" >
        <item>
         <property name="text" >
          <string>Use Normalized Name</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Use Literal Name</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="qualityLabel" >
        <property name="text" >
         <string>Quality:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KComboBox" name="qualityComboBox" >
        <item>
         <property name="text" >
          <string>Best Quality (.mp4)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Normal Quality (.flv)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="loginGroupBox" >
     <property name="toolTip" >
      <string>Provide login info of your YouTube account in order to access restricted media.</string>
     </property>
     <property name="title" >
      <string>Login Info</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QFormLayout" name="formLayout" >
        <item row="2" column="0" >
         <widget class="QLabel" name="passwordLabel" >
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="KLineEdit" name="passwordEdit" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="usernameLabel" >
          <property name="text" >
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="KLineEdit" name="usernameEdit" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="useNetrcCheck" >
        <property name="toolTip" >
         <string>.netrc must have a hostname called 'youtube'.</string>
        </property>
        <property name="text" >
         <string>User .netrc file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
